crowd_anki/importer/anki_importer.py

```diff
@@ -42,7 +42,7 @@ def load_from_directory(self, directory_path, import_media=True):
             aqt.mw.progress.start(immediate=True)
 
         try:
-            self.load_from_file(directory_path.joinpath(directory_path.name).with_suffix(DECK_FILE_EXTENSION))
+            self.load_from_file(directory_path.joinpath('deck').with_suffix(DECK_FILE_EXTENSION))
```

Comment:
You should update the docstring of the function to describe new convention

Reply:
done in the latest commit.